client/verta/tests/unit_tests/test_endpoint_class.py

```diff
@@ -0,0 +1,145 @@
+# -*- coding: utf-8 -*-
+# Unit tests for the Endpoint class.
+#
+#  Mocked API responses are declared as static test fixtures for now.  This is a brittle approach, as any schema changes
+#  in those calls will not be reflected here unless manually updated.  Long term, we'll want to generate these fixtures
+#  dynamically from a shared schema, like swagger.  Additionally, some of the patching of methods is a tad cumbersome.
+#  We'll want to consider a less verbose approach for patching in the future (decorators).
+
+import os
+import pytest
+
+from copy import deepcopy
+from typing import Dict, Any
+from unittest.mock import patch
+
+from client.verta.verta.endpoint import Endpoint
+from client.verta.verta._internal_utils._utils import Connection, Configuration
+from client.verta.verta.credentials import EmailCredentials
```

Comment:
Would you be opposed to importing from `verta` instead?

```suggestion
from verta.endpoint import Endpoint
from verta._internal_utils._utils import Connection, Configuration
from verta.credentials import EmailCredentials
```

Our tests in ci are executed [from the `client/verta/tests/` directory](https://github.com/VertaAI/cluster-setup/blob/aecb4b8ea191b18b2fbedc4fb6938d1ae754db01/ci/client/python/pytest-oss-3.10.Jenkinsfile#L67) so this `import` wouldn't work (and I'm not sure if there's a reason to require that `pytest` be run from the root of the repository).